build/readme-parser.ts

```diff
@@ -53,7 +57,7 @@ export function getFeaturesMeta(): FeatureMeta[] {
 }
 
 export function getFeatures(): FeatureID[] {
-	const contents = readFileSync(path.join(__dirname, '../source/refined-github.ts'), 'utf-8');
+	const contents = getImportedFeatures();
```

Comment:
I just realized that `getFeatures` already parses the whole file so you can just use that directly instead of creating `getImportedFeatures`